### PR TITLE
added Series Eleven Cryo Drones

### PR DESCRIPTION
**Pull request by bleehu**
I'm a little concerned that there's a lot more instances in the game of heat damage than cold damage, thus the new Cryo Drone is underpowered in general, if not in the specific case that they're in their element. 

UV light in forensics is a pretty real thing. The science there was fairly well thought out.

### CODE REVIEW

**Review by Reveraine — APPROVED** [review]
I like them! Ship it!
Also, Cryo-drones would also function perhaps better in open space than other drones?

**Comment by bleehu** [comment]
I was thinking that too!

**Comment by Turtlelord26** [comment]
Functioning well in a vacuum would go a pretty good way towards redressing that concern about the prevalence of fire damage, methinks.

**Comment by bleehu** [comment]
I added some serious space mobility bonuses.

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 1]
```diff
@@ -164,3 +164,53 @@ However, these systems are not well-calibrated for extreme cold. Long exposure
...
+be normally habitable by most organics and standard series Drones. These Drones
+range from the stalwart soldiers and workers who defend and move the heavy 
+machinery needed to convert ice into usable water to the light and slim covert
+operators that infiltrate distant bases on snowy planets. Most Series Eleven
+Drones have patterns of white, blue, black and purple exo plates. They also have
+a distinct kind of synthetic fur that slowly grows along their joints to help
+slow their rate of cooling.
+
+Size: Medium
+Favored Terrain: Subzero Mountains and Tundra
```
Comment (Turtlelord26) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:181]: Suggest also Barren Rock and Spacewalk

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 2]
```diff
@@ -164,3 +164,53 @@ However, these systems are not well-calibrated for extreme cold. Long exposure
...
+seen through smoke or in no-light conditions. But enemies without lasers, 
+shields, or plasma weapons will be hidden by darkness or smoke. 
+
+Cold Never Bothered Me:
+    Being able to retain heat and stay insulated makes Cryo Series Drones able
+to withstand insane cold almost indefinitely. Unfortunately, this means that
+the Cryo Series has a hard time radiating heat away, and also the insulating
+artificial fur doesn't do well when submerged in water. 
+
+    Drones unfettered by cold are immune to damage from cold and similar
```
Comment (Turtlelord26) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:209]: Straight up immunity concerns me from a balance perspective. If it's important to the balance of this trait, the drawback to lessen would I think be the plasma persistence (from 3 turns to 2).
Strictly speaking, nonblocking.

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 3]
```diff
@@ -164,3 +164,53 @@ However, these systems are not well-calibrated for extreme cold. Long exposure
...
+to withstand insane cold almost indefinitely. Unfortunately, this means that
+the Cryo Series has a hard time radiating heat away, and also the insulating
+artificial fur doesn't do well when submerged in water. 
+
+    Drones unfettered by cold are immune to damage from cold and similar 
+environmental conditions. The insulations also help to dampen sound and obscure
+the Drone's silhouette, giving a +10 to stealth skill checks. Being adept at
+operating in cold environments has led to Cryo Drones being better at operating
+in space and in zero gravity. Cryo Drones get a +20 to Reflex saves to move in
+zero grav and +20 to Accrobatics checks to maneuver in zero gravity as well.
```
Comment (Turtlelord26) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:214]: I do appreciate that the Reflex bonus indicates someone read my microgravity movement rules :)

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**